app/views/BookDetail/index.tsx

```diff
@@ -156,25 +155,29 @@ mutation DeleteCartItem($id: ID!) {
 `;
 
 function BookDetail() {
-    const { id } = useParams();
+    // const { id } = useParams<{ id: string }>();
```

Comment:
remove this line

Reply:
Let's not remove that line. It is how the function should be used.
It's kept as a reference for the time when typings will be fixed